### PR TITLE
docs: some improvements

### LINKED ISSUE #128: Improve docs
- [x] Better flow for different types of developers in README.md
- [x] Better explain my-inway name
- [x] Absolute path between quotes in building an application, creating a service
- [x] Make supported API's more explicit
- [x] Something about Docker networking related to host in creating a service
- [x] Section about logging
- [x] Add documentation for new features: `x-nlx-request-organization`, `x-nlx-request-logrecord-id`

### LINKED ISSUE #136: Add warning to service configuration file
A warning in header of this file might be wise, something like:

  W A R N I N G !
The currently deployed online NLX network is for demo purposes and not ready  for connected resources containing sensitive data.
When using real personal data, use your own NLX network in an environment you control

https://github.com/VNG-Realisatie/nlx/blob/be0b78d27a7bfa386fa31ca9df2dcb686b89e9af/inway/service-config.toml#L4

### PR DESCRIPTION
**Pull request by bartjkdp**
This adds some explanation to the documentation and README.md and closes #128 and closes #136.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +34,6 @@ cd nlx
 docker-compose up
 ```
 
-## Licence
-
-Copyright © VNG Realisatie 2017  
+## License
+Copyright © VNG Realisatie 2017
```
Comment (bvhme) [inline README.md]: Trailling whitespace here is MarkDown syntax for the `<br>` element, without it this line and the next line will be rendered on as the same line in a paragraph, in stead of two lines in a paragraph.

File: docs/content/introduction/logging.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* `X-NLX-Request-Application-Id`, the id of the application performing the request
+* `X-NLX-Request-Subject-Identifier`, an subject identifier for purpose registration (doelbinding)
+* `X-NLX-Request-Process-Id`, a process id for purpose registration (doelbinding)
+* `X-NLX-Request-Data-Elements`, a list of requested data elements
+
+The outway appends a globally unique `X-NLX-Request-Id` to make a request traceable through the network. All the headers are logged before the request leaves the outway. Then the fields `X-NLX-Request-User-Id`, `X-NLX-Request-Application-Id`, and `X-NLX-Request-Subject-Identifier` are stripped of and the request is forwarded to the inway. When a request arrives at the inway, the inway logs the request and also the requesting organization.
+
+Logging is done to stdout in JSON format. You can use for example [fluentd](https://www.fluentd.org/) to collect the logs from the containers and forward them to a store. This is an example configuration for fluentd when using Kubernetes:
+
+```
```
Comment (bvhme) [inline docs/content/introduction/logging.md]: Consider either using [GFM fenced code blocks](https://github.github.com/gfm/#fenced-code-blocks/) ` ``` `, preferably with a language statement (like you've done in the other files), or using MarkDown's `    ` [four space code block signifier](https://daringfireball.net/projects/markdown/syntax#precode), and not both (unless the code you are actually needs to be indented by 4 spaces)

Comment (bartjkdp) [reply docs/content/introduction/logging.md]: What exactly would you place between the code blocks? I think I am only referring to variable names.

File: README.md (hunk 1 above)
Comment (bartjkdp) [reply README.md]: Whoops, my bad.

**Comment by bartjkdp** [comment]
Thanks a lot for the review!

1. Instructions for setting up local NLX are still correct.
2. Not following a specific style guide. I find this more readable and in this way it is consistent with the formatting we use elsewhere in the codebase.

File: docs/content/introduction/logging.md (hunk 2 above)
Comment (bvhme) [reply docs/content/introduction/logging.md]: I mean the code block that is underneath this comment, it is both fenced with ```` ``` ```` and with 4 spaces ```    ```

**Comment by bvhme** [comment]
Cool, as long as it is consistent, I just don't offen see that, probably because although the standard explicitly allows it I also know it throws some processors.

👍

**Comment by bvhme** [comment]
Hey, is there something I can help with to merge this ASAP?

File: docs/content/introduction/logging.md (hunk 2 above)
Comment (bvhme) [reply docs/content/introduction/logging.md]: So, is this intended?

Comment (bartjkdp) [reply docs/content/introduction/logging.md]: Now I get it, fixed now :)

**Pull request merged**